sfml/squash_game/Game.cpp

```diff
@@ -0,0 +1,118 @@
+#include <SFML/Graphics.hpp>
+#include "TextException.h"
+#include <math.h>
+#include "Game.h"
+#include "Utils.h"
+#include "Ball.h"
+#include "Racquet.h"
+#include <memory>
+
+
+Game::Game(sf::RenderWindow& window) 
+{
+  using namespace Utils;
+
+  objects.push_back(std::unique_ptr<Ball> (new Ball(int(window_x / 2 - ball_r), ball_start_height, ball_r, ball_angle, ball_speed)));
```

Comment:
I am not actually sure what exactly do you mean, there could be different arrays for different solutions